src/vivarium_public_health/disease/special_disease.py

```diff
@@ -94,13 +98,25 @@ def __init__(self, cause, risk):
         self.configuration_defaults = {
             self.cause.name: RiskAttributableDisease.configuration_defaults['risk_attributable_disease']
         }
+        self._state_names = [f'{self.cause.name}', f'susceptible_to_{self.cause.name}']
+        self._transition_names = [f'susceptible_to_{self.cause.name}_TO_{self.cause.name}',
+                                    f'{self.cause.name}_TO_susceptible_to_{self.cause.name}']
```

Comment:
The second transition here should only exist if the disease is recoverable, which you can find in the configuration